### DISCUSSION TITLE
Emgu on Raspberrypi error - global cap_v4l.cpp:1134 tryIoctl

### DISCUSSION THREAD

**Discussion by D3D-s** [emgucv/emgucv#930] [Q&A]
Hello,

I have Raspberry 4B 64 bit. I followed all the instructions for installation and am trying to execute a simple code.
using (VideoCapture camera = new VideoCapture(0))
{
    Mat image = new Mat();
    bool status = camera.Read(image); <--error here

    //or image = camera.QueryFrame();
    if (!image.IsEmpty)
    {
        Console.WriteLine("super");
    }

}
 after few seconds of delay on this line 
 bool status = camera.Read(image); 
 
 I get the following error. Please help!!!

[ WARN:0@10.033] global cap_v4l.cpp:1134 tryIoctl VIDEOIO(V4L2:/dev/video0): select() timeout.
[ WARN:1@10.036] global cap_v4l.cpp:1134 tryIoctl VIDEOIO(V4L2:/dev/video0): select() timeout.

**Comment by emgucv** [comment]
Raspberry pi's camera module is not compatible with Open CV's VideoCapture class. 

This is the Open CV ticket if you want to follow:
https://github.com/opencv/opencv/issues/21653
